Missing logs of element not found when using click(), getText() inside PageObject with async/await

## Describe the bug
<!--
A clear and concise description of what the bug is, how to reproduce it, and what the expected behaviour is.

Please make sure to format the code samples, verbose output, configuration and other inline debug information (use the Preview functionality). Check out the [GitHub Markdown guide](https://guides.github.com/features/mastering-markdown/).
-->
Probably I'll get the "Golden Shovel" award  for digging up such issues but, what the heck. I've notice that there are no logs about unsuccessful click() or getText() and probably other methods when they are called inside Page Object command with ES6 async/await.

## Sample test
<!-- 
Include a sample test which reproduces the problem you're experiencing. If possible, the test should be against a public url.

Please add the test and other info inline, not as attachments or screenshots.
-->

<details><summary>sampleTest.js</summary><p>
<!-- browsers demand the next line be empty -->

```js
// Please add the sample test here

module.exports = {
    "Dummy async": async function (browser: NightwatchBrowser) {
        const pageObject = browser.page.testPage();
        await browser.url('https://www.ecosia.org/')
        await pageObject.dummyAsyncCommand();
    }
};
```
</details>
<details><summary>simplePageObject.js</summary><p>
<!-- browsers demand the next line be empty -->

```js
const testCommands = {

    dummyChainedCommand: function () {
        return this.click("#IdForElementThatDoesntExists")
            .click("#IdForElementThatDoesntExists")
            .waitForElementVisible('#IdForElementThatDoesntExists')
    },
    dummyAsyncCommand: function () {
        return new Promise(async (resolve) => {
            await this.click("#IdForElementThatDoesntExists");
            await this.getText('#IdForElementThatDoesntExists');
            await this.waitForElementVisible('#IdForElementThatDoesntExists');
            resolve();
        })
    }
}

const testPage = {
    url: 'https://nightwatchjs.org/',
    elements: {
        cssSelector: '#someCssSelector',
        cssSelectorObject: {
            selector: '#someSelectorObjectSelector',
            locateStrategy: 'css selector'
        },
        xPathSelector: '//*[@id="someXpathSelector"]',
        xPathSelectorObject: {
            selector: '#someSelectorObjectSelector',
            locateStrategy: 'xpath'
        }
    },

    commands: testCommands,
    props: {},
    sections: {}
};

module.exports = testPage;
```
</p></details>

**Run with command**
<!-- Include the command used to run the test, if applicable. -->

```sh
$ nightwatch test/sampleTest.js --your-other-arguments-here
```

### Verbose output
<details><summary>debug.log</summary><p>
<!-- browsers demand the next line be empty -->

```txt
[Error When Element Not Found] Test Suite
=========================================
\ Connecting to localhost on port 9515...

i Connected to localhost on port 9515 (2711ms).
  Using: chrome (86.0.4240.111) on Windows platform.

Running:  Dummy async

× Timed out while waiting for element <#IdForElementThatDoesntExists> to be present for 5000 milliseconds. - expected "visible" but got: "not found" (5232ms)
    at Promise (C:\Users\admin\Documents\Repo\CS\NightwatchPlayground\Playground\pages\testPage.js:11:24) 


FAILED: 1 assertions failed and  2 errors (16.805s)
_________________________________________________

TEST FAILURE: 1 error during execution; 0 tests failed, 0 passed (19.791s)

  NightwatchAssertError: Timed out while waiting for element <#IdForElementThatDoesntExists> to be present for 5000 milliseconds. - expected "visible" but got: "not found" (5232ms)
   


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! playground@1.0.0 e2e: `tsc && nightwatch ./tests/errorWhenElementNotFound.js --env chrome --url`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the playground@1.0.0 e2e script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\admin\AppData\Roaming\npm-cache\_logs\2020-10-28T09_46_16_181Z-debug.log
```
</p></details>


## Configuration
<details><summary>nightwatch.json</summary><p>
<!-- browsers demand the next line be empty -->

<!-- Please paste your nightwatch.json or nightwatch.conf.js here; make sure to leave out any sensitive details -->

```js
{
module.exports = {
    // An array of folders (excluding subfolders) where your tests are located;
    // if this is not specified, the test source must be passed as the second argument to the test runner.

    //src_folders: ['./tests'],

    // See https://nightwatchjs.org/guide/working-with-page-objects/
    page_objects_path: './pages',

    // See https://nightwatchjs.org/guide/extending-nightwatch/#writing-custom-commands
    custom_commands_path: './node_modules/*****/commands',

    // See https://nightwatchjs.org/guide/extending-nightwatch/#writing-custom-assertions
    custom_assertions_path: './node_modules/******/assertions',

    // See https://nightwatchjs.org/guide/#external-globals
    globals_path: './globals/globalsModule.js',

    webdriver: {
        start_process: true
    },
    live_output: true,
    disable_error_log: false,
    parallel_process_delay: 1000,
    test_settings: {
        firefox: {
            skiptags: 'chrome',
            screenshots: {
                enabled: true,
                path: 'screens_firefox',
                on_failure: true,
                on_error: true
            },
            desiredCapabilities: {
                browserName: 'firefox',
                alwaysMatch: {
                    'moz:firefoxOptions': {,
                        args: []
                    }
                }
            },
            webdriver: {
                port: 4444,
                server_path: Services.geckodriver ? Services.geckodriver.path : '',
                cli_args: []
            }
        },
        firefox_cli: {
            skiptags: 'chrome',
            screenshots: {
                enabled: true,
                path: 'screens_firefox',
                on_failure: true,
                on_error: true
            },
            desiredCapabilities: {
                browserName: 'firefox',
                alwaysMatch: {
                    'moz:firefoxOptions': {
                        args: [
                            '-headless'
                        ]
                    }
                }
            },
            webdriver: {
                port: 4444,
                server_path: Services.geckodriver ? Services.geckodriver.path : '',
                cli_args: []
            }
        },
        chrome: {
            skiptags: 'firefox',
            screenshots: {
                enabled: true,
                path: 'screens_chrome',
                on_failure: true,
                on_error: true
            },
            desiredCapabilities: {
                browserName: 'chrome',
                chromeOptions: {
                    prefs : { 
                        download: {
                          prompt_for_download: false,
                          default_directory: downloadsChromePath
                        },
                        profile: {
                          default_content_setting_values : { 
                            automatic_downloads: 1
                            }
                        }
                      },
                    args: []
                }
            },
            webdriver: {
                port: 9515,
                server_path: Services.chromedriver ? Services.chromedriver.path : '',
                cli_args: []
            }
        },
        chrome_cli: {
            skiptags: 'firefox',
            screenshots: {
                enabled: true,
                path: 'screens_chrome',
                on_failure: true,
                on_error: true
            },
            desiredCapabilities: {
                browserName: 'chrome',
                chromeOptions: {
                    prefs : { 
                        download: {
                          prompt_for_download: false,
                          default_directory: downloadsChromePath
                        },
                        profile: {
                          default_content_setting_values : { 
                            automatic_downloads: 1
                            }
                        }
                      },
                    args: [
                        '--headless'
                    ]
                }
            },
            webdriver: {
                port: 9515,
                server_path: Services.chromedriver ? Services.chromedriver.path : '',
                cli_args: []
            }
        },
        chrome_networkTraffic: {
            screenshots: {
                enabled: true,
                path: 'screens_chrome',
                on_failure: true,
                on_error: true
            },
            desiredCapabilities: {
                browserName: 'chrome',
                chromeOptions: {
                    args: [
                        '--ignore-certificate-errors',
                        ]
                },
                // browsermob-proxy
                proxy: {
                    proxyType: 'manual',
                    sslProxy: 'localhost'
                },
            },
            webdriver: {
                port: 9515,
                server_path: Services.chromedriver ? Services.chromedriver.path : '',
                cli_args: []
            }
        }
    }
};
}

globals:
module.exports = {
    'default': {
        isLocal: true
    },
    'integration': {
        isLocal: false
    },
    backendData: {}, //object for data from BackEnd

    before: function (browser, done) {
        done();
    },
    // This will be run before each test suite is started
    beforeEach: function (browser, done) {
        browser.status(function (result) {
            done();
        });
    },
    // This will be run after each test suite is finished
    afterEach: function (browser, done) {
        done();
    }
};
```
</p></details>

## Your Environment
<!--- Include the relevant details your environment -->

| Executable | Version |
| ---: | :--- |
| `nightwatch --version` | 1.4.3 |
| `npm --version`  | VERSION |
| `yarn --version` | VERSION |
| `node --version` | VERSION |

| Browser driver | Version |
| --- | --- |
| NAME | VERSION |
| chromedriver | 86 |

<!-- For example:
| geckodriver | v0.25.0 |
| selenium-server | 3.141 |
-->

| OS | Version |
| --- | --- |
| NAME | VERSION |
<!-- For example:
| macOS Sierra | 10.12.3 |
| Windows 10 | 1607 |
| Ubuntu | 16.10 |
-->

--- a/lib/api/element-commands/_baseElementCommand.js
+++ b/lib/api/element-commands/_baseElementCommand.js
@@ -88,6 +88,8 @@ class BaseElementCommand extends ElementCommand {
     };
 
     if (shouldRegister) {
+      Utils.Logger.error(err);
+
       this.reporter.registerTestError(err);
       err.registered = true;
     }

